zerver/lib/message.py

```diff
@@ -149,23 +151,28 @@ def sew_messages_and_submessages(messages: List[Dict[str, Any]],
             message = message_dict[message_id]
             message['submessages'].append(submessage)
 
+
 def extract_message_dict(message_bytes: bytes) -> Dict[str, Any]:
     return ujson.loads(zlib.decompress(message_bytes).decode("utf-8"))
 
+
 def stringify_message_dict(message_dict: Dict[str, Any]) -> bytes:
     return zlib.compress(ujson.dumps(message_dict).encode())
 
-@cache_with_key(to_dict_cache_key, timeout=3600*24)
+
+@cache_with_key(to_dict_cache_key, timeout=3600 * 24)
 def message_to_dict_json(message: Message) -> bytes:
     return MessageDict.to_dict_uncached(message)
 
+
```

Comment:
Can you reconfigure your editor to not automatically add these newlines and otherwise reformat the code?  They are not part of our style and make your changes very hard to read.

Reply:
Okay 
I will look into it

Reply:
There was no mention of it in the `lint` test or the [documentation](https://zulip.readthedocs.io/en/latest/contributing/code-style.html#be-consistent). I will update the docs once I am done with this PR.